history/cultures/alteraci.txt

```diff
@@ -1,51 +1,75 @@
 1.1.1 = {
```

Comment:
I mean you really need to check and move all the cultures so it makes sense.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/124789313-3d655080-df5b-11eb-94fd-7d726280f60a.png)
![image](https://user-images.githubusercontent.com/46384992/124789333-40f8d780-df5b-11eb-9fc0-ed2f131cf0fc.png)
![image](https://user-images.githubusercontent.com/46384992/124789361-45bd8b80-df5b-11eb-9bc8-fa78bea90d9c.png)
![image](https://user-images.githubusercontent.com/46384992/124789380-48b87c00-df5b-11eb-8e61-075f643849a8.png)

</details>